charts/azure/azure-disk-loss/experiment.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    Detaching the disk attached to azure vm instance
```

Comment:
Can we change "Detaching" to "Detaches"? Also a bit more information can be added, such as the disk gets re-attached at the end of the experiment and that the disk is supposed to be not a boot-disk.